.github/workflows/main.yml

```diff
@@ -52,7 +53,7 @@ jobs:
           github.repository == 'scikit-fuzzy/scikit-fuzzy' &&
           github.ref == 'ref/head/master' &&
           matrix.os == 'ubuntu-latest' &&
-          matrix.python == 3.9
+          matrix.python == 3.x
```

Comment:
This produces an invalid workflow from GH Actions, I think the ideal course is to specify `3.11` to only deploy docs from the latest stable release.  Otherwise looks good!

Reply:
> This produces an invalid workflow from GH Actions

What?!?  See the section on `x-ranges`.  https://github.com/actions/setup-python/blob/main/docs/advanced-usage.md#using-the-python-version-input

Reply:
Not sure why, perhaps because it's in the if-statement, but here's GH Actions' complaint - let me know if I misinterpreted!
![image](https://github.com/scikit-fuzzy/scikit-fuzzy/assets/2184487/2dccc6c0-1e8a-41c2-9880-08a84b1a94f4)

Reply:
I reverted the change so we should be good.